pypeit/edgetrace.py

```diff
@@ -877,11 +877,19 @@ def auto_trace(self, bpm=None, debug=False, show_stages=False):
         if not self.is_empty and self.par['add_missed_orders']:
             # Refine the order traces
             self.order_refine(debug=debug)
-            # Check that the edges are still sinked (not overkill if orders are
-            # missed)
-            self.success = self.sync()
-            if not self.success:
-                return
+            # Check that the edges are still sinked
+            if not self.is_synced:
+                msgs.error('Traces are no longer synced after adding in missed orders.')
+
+#           KBW: This was the old code that resynced.  The comment suggests
+#           this may be necessary if orders are missed, so I need to test if
+#           that's true.
+#            # Check that the edges are still sinked (not overkill if orders are
+#            # missed)
+#            self.success = self.sync(debug=True)
+#            if not self.success:
+#                return
+
```

Comment:
Can the above commented out code be removed or is there still more testing needed?

Reply:
I'd like to keep it around for a while.  I've improved the comment a bit to indicate this.